ctrd/containerd.go

```diff
@@ -246,3 +246,27 @@ func (ctrd *Ctrd) RemoveSnapshot(ctx context.Context, id string) error {
 	return service.Remove(ctx, id)
 
 }
+
+// NewContainer just create a very simple container for migration
+// just load container id to boltdb
+func (ctrd *Ctrd) NewContainer(ctx context.Context, id string) error {
+	ctx = namespaces.WithNamespace(ctx, namespaces.Default)
+
+	spec, err := oci.GenerateSpec(ctx, nil, &containers.Container{ID: id})
```

Comment:
Do we need to remove the containerID info if any step failed?

Reply:
make sense, i will fix